tools/cdn.sh

```diff
@@ -1,27 +1,40 @@
-CURRENT_VERSION=$(node tools/attribute.js version)
+VERSION="version"
+MAJORMINOR="majorminor"
+
+CURRENT_VERSION=$(node tools/attribute.js $VERSION)
 EXTENSION_NAME="auth0-account-link"
+PACKAGE_JSON='../package.json'
+MAJORMINOR_VERSION=$(node tools/attribute.js $VERSION $PACKAGE_JSON $MAJORMINOR)
+
+deploy_bundle() {
```

Comment:
nit: could check that `$1` is a non-empty string here, and error out if it is

Reply:
Yeah I'll add that in

Reply:
Added in check